scripts/generate_coverage/run_python.py

```
@@ -54,10 +66,23 @@ def main(
         raise typer.Exit(code=exc.retcode or 1) from exc
 
     if fmt == "coveragepy":
+        xml_tmp = out.with_suffix(".xml")
+        try:
+            python["-m", "coverage", "xml", "-o", str(xml_tmp)]()
+        except ProcessExecutionError as exc:
+            typer.echo(
+                f"coverage xml failed with code {exc.retcode}: {exc.stderr}", err=True
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)